<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaceDialog</class>
 <widget class="QDialog" name="PlaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>اعدادات</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Arabic" country="SaudiArabia"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>الدولة والمدينة</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QListWidget" name="citiesListWidget">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QListWidget" name="countriesListWidget">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>بيانات</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>خط الطول:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="longitudeLineEdit">
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>خط العرض:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="latitudeLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>فرق التوقيت عن غرينيتش :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="zoneLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>252</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>خيارات</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>المذهب :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="mazhabComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>الشافعي،الحنبلي،المالكي</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>الحنفي</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>149</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>التوقيت</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QRadioButton" name="winterRadioButton">
           <property name="text">
            <string>شتوي</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="summerRadioButton">
           <property name="text">
            <string> صيفي</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>التقويم :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="calenderComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>جامعة ام القرى</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>الهيئة المصرية للمساحة</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>جامعة العلوم الاسلامية بكراتشي</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>الاتحاد الاسلامي بامريكا الشمالية</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>رابطة العالم الاسلامي</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>موافق</string>
       </property>
       <property name="icon">
        <iconset resource="gui.qrc">
         <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>الغاء</string>
       </property>
       <property name="icon">
        <iconset resource="gui.qrc">
         <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>348</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections/>
</ui>
